### ISSUE TITLE
Fix unclickable download button for GroupCard in the dashboard

### ISSUE THREAD

**Issue by danmaninc** [one-zero-eight/InNoHassle-Website#11]
![Image](https://github.com/one-zero-eight/InNoHassle-Website/assets/71718152/0e5399fb-2281-4cbd-ac2a-97909a357b9e)

**Comment by ArtemSBulgakov** [comment]
Maybe use [intercepting routes](https://nextjs.org/docs/app/building-your-application/routing/intercepting-routes) in Next.js to display dialog when navigating to URL of the group

**Comment by danmaninc** [comment]
> Maybe use [intercepting routes](https://nextjs.org/docs/app/building-your-application/routing/intercepting-routes) in Next.js to display dialog when navigating to URL of the group

This is good idea. However, this requires availability of _page for a single GroupCard_.
Why?

Current modal is opened by using hooks in wrapping element (for example, ScheduleElement modal opens due to the hooks in ScheduleList). And such system does not use any routes. 
So I have committed temporary fix, using the same system (https://github.com/one-zero-eight/InNoHassle-Website/pull/29/commits/099e4e31116ea67ebc8c21ead0caf77b1dfa7aec)

Also, I have opened #34, which includes the usage of intercepting routes.
